cdc/model/sink.go

```diff
@@ -18,21 +18,31 @@ const (
 	MqMessageTypeResolved
 )
 
+// TableName represents name of a table, includes table name and schema name.
+type TableName struct {
+	Schema string `json:"shema"`
+	Table  string `json:"table"`
+}
+
 // RowChangedEvent represents a row changed event
 type RowChangedEvent struct {
-	Ts    uint64
-	RowID int64
+	StartTs uint64 `json:"start-ts"`
 
-	Schema string
-	Table  string
+	// Commit or resolved TS
+	CRTs     uint64 `json:"cr-ts"`
+	Resolved bool   `json:"resolved"`
```

Comment:
in new implement of sink, we don't need `Resolved ` field in `RowChangedEvent ` any more